### DISCUSSION TITLE
Welcome to UCUM discussions!

### DISCUSSION THREAD

**Discussion by ucumworker** [ucum-org/ucum#183] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together.

**Comment by aflackey** [comment]
Thanks @ucumworker. Are these discussions private to our UCUM community, or open to the full GitHub community?

**Reply by timbrisc** [reply]
They can be either.
